oauth/oauth.cabal

```diff
@@ -5,7 +5,7 @@ cabal-version: 1.12
 -- see: https://github.com/sol/hpack
 
 name:           oauth
-version:        0.0.3
+version:        0.0.4
 synopsis:       Haskell Oauth library
 description:    Haskell Oauth library currently supporting only google auth
```

Comment:
Comment: @mbj The OAuth library has morphed in a way that makes it more generic than what the current description implies. I think the description update is in order. 
Is there an actual spec for this OAuth flow where the request and responses match what we have?

Reply:
most implement a super / subset. So we'll implement the subset of the providers we have in production.

Can you PR an update, good catch BTW.

Reply:
> Can you PR an update, good catch BTW.

Sure, let me do so.